Publication of LOOPS book 1 by Stephen Kaisler

pamoroso opened the pull request:
This change lists on the [Documentation](https://interlisp.org/software/using-medley#interlisp-books) page the LOOPS books by Stephen Kaisler and links to the first volume.

pamoroso committed:
Fix typos

pamoroso committed:
Merge branch 'Interlisp:main' into main

pamoroso committed:
Initial commit

Add the book Medley LOOPS: The Basic System.

pamoroso committed:
List LOOPS books and link the first

List the LOOPS books by Stephen Kaisler and link to the first volume.

masinter reviewed (approved):
sh0uld this say "was an object extension of Interlisp" instead of "is"??

Somewhere there should be a mention of its role compared to CLOS and CommonLoops?
Possibly somewhere else, but raise that as an issue if you agree.

pamoroso committed:
Change verb tens

Change verb tens to highlight that LOOPS was the object extension of Interlisp.

pamoroso committed:
The object extension -> an object extension

pamoroso commented:
I changed to "was an object extension of Interlisp".

pamoroso commented:
Concerning the role of LOOPS I raised [issue #1850](https://github.com/Interlisp/medley/issues/1850).

stumbo commented on static/documentation/2024-loops-book-1.pdf.pdf:
Rename to `static/documentation/2024-loops-book-1.pdf`  -- remove the extra `.pdf`

pamoroso replied:
Oops... Good catch, I didn't realize. Fixed.

pamoroso committed:
Delete incorrectly named file

Delete the file to reupload it with correct name.

pamoroso committed:
Rename file

Upload file with the correct name.
